### PR TITLE
[typo](docs) modify bitmap function document

### PR DESCRIPTION
**Pull request by vinlee19**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/sql-manual/sql-functions/bitmap-functions/bitmap_or.md [hunk 1]
```diff
@@ -35,42 +35,49 @@ Compute union of two or more input bitmaps, returns the new bitmap.
...
-MySQL> select bitmap_to_string(bitmap_or(to_bitmap(1), to_bitmap(2)));
+mysql> select bitmap_to_string(bitmap_or(to_bitmap(1), to_bitmap(2)));
 +---------------------------------------------------------+
 | bitmap_to_string(bitmap_or(to_bitmap(1), to_bitmap(2))) |
 +---------------------------------------------------------+
 | 1,2                                                     |
 +---------------------------------------------------------+
 
-MySQL> select bitmap_to_string(bitmap_or(to_bitmap(1), to_bitmap(2), to_bitmap(10), to_bitmap(0), NULL));
+mysql> select bitmap_to_string(bitmap_or(to_bitmap(1), to_bitmap(2), to_bitmap(10), to_bitmap(0), NULL));
```
Comment (jacktengg) [inline docs/en/docs/sql-manual/sql-functions/bitmap-functions/bitmap_or.md:66]: The reslt of this example is wrong, it should be:
```
mysql [(none)]>select bitmap_to_string(bitmap_or(to_bitmap(1), to_bitmap(2), to_bitmap(10), to_bitmap(0), NULL));
+--------------------------------------------------------------------------------------------+
| bitmap_to_string(bitmap_or(to_bitmap(1), to_bitmap(2), to_bitmap(10), to_bitmap(0), NULL)) |
+--------------------------------------------------------------------------------------------+
| 0,1,2,10                                                                                   |
+--------------------------------------------------------------------------------------------+
1 row in set (0.03 sec)
```

**Review by jacktengg — APPROVED** [review]
LGTM

**Review by jacktengg — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run buildall

**Review by zy-kkk — APPROVED** [review]

**Comment by vinlee19** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**